tests/hmc/Test_hmc_IwasakiGauge_expected.txt

```diff
@@ -0,0 +1 @@
+8.8.8.8 1.1.1.1 0.269298793 633bf471 3a22ad20
```

Comment:
Those two expected values files have to be adjusted for the extra test parameters and line numbers. @qiUip